manifests/jenkins-with-cert.yaml

```diff
@@ -20,7 +20,11 @@ objects:
           FROM overridden
           COPY cert/data /etc/pki/ca-trust/source/anchors/root-ca.crt
           USER root
-          RUN update-ca-trust
+          RUN if grep 'dummy' /etc/pki/ca-trust/source/anchors/root-ca.crt; then \
+                  rm /etc/pki/ca-trust/source/anchors/root-ca.crt; \
+              else \
+                  update-ca-trust; \
+              fi
```

Comment:
Did you try the `EOF` ? I know it works in newer podman build, but maybe OpenShift builds it doesn't work yet?

```
RUN <<EOF
    if
    else
    fi
EOF
```

Reply:
Yes, it doesn't seem to work

Reply:
Grr.. It's such a nice feature!